crates/uv/src/commands/project/lock.rs

```diff
@@ -223,13 +223,13 @@ pub(super) async fn do_lock(
         if requires_python.is_unbounded() {
             let default =
                 RequiresPython::greater_than_equal_version(&interpreter.python_minor_version());
-            warn_user!("The workspace `requires-python` field does not contain a lower bound: `{requires_python}`. Set a lower bound to indicate the minimum compatible Python version (e.g., `{default}`).");
+            warn_user!("The workspace `Requires-Python` field does not contain a lower bound: `{requires_python}`. Set a lower bound to indicate the minimum compatible Python version (e.g., `{default}`).");
```

Comment:
The problem is that toml is case sensitive, so `Requires-Python = ">=3.8"` does not.

Reply:
@konstin is probably right on that one.

Reply:
Tragic.

Reply:
I know, aesthetically I dislike it.

Reply:
This is a far more invasive change if we want to be consistent throughout our internal comments as well.